vilebot/src/main/java/com/oldterns/vilebot/handlers/user/Kaomoji.java

```diff
@@ -55,6 +55,10 @@ private String kaomojiify(String message) throws Exception {
             else if (words[0].equals("nsfw") | words[0].equals("wtf")){
```

Comment:
I think circuit-break or `||`  is intended here instead of bitwise or `|`. Since both operands are booleans, the only different is that this might do an additional `equals` call that is not needed.

Reply:
You’re right, that change would be preferable. I might as well change it in this PR.